crypto/cmp/cmp_server.c

```diff
@@ -147,6 +162,25 @@ int OSSL_CMP_SRV_CTX_set_grant_implicit_confirm(OSSL_CMP_SRV_CTX *srv_ctx,
     return 1;
 }
 
+static OSSL_CMP_MSG *delayed_delivery(OSSL_CMP_SRV_CTX *srv_ctx,
```

Comment:
Please prepend a brief documenting comment what this local function expects and returns.

Reply:
Slightly better: `Return error msg with waiting status if polling is initiated, else NULL */`